tox.ini

```diff
@@ -16,4 +16,4 @@ deps =
     py{27,33,34,35}: flake8-import-order>=0.9
     nose>=1.3.1
     py26: unittest2>=0.5.1
-passenv = BIOBLEND_GALAXY_URL BIOBLEND_GALAXY_API_KEY GALAXY_VERSION
+passenv = BIOBLEND_GALAXY_URL BIOBLEND_GALAXY_API_KEY BIOBLEND_GALAXY_MASTER_API_KEY BIOBLEND_GALAXY_USER_EMAIL GALAXY_VERSION
```

Comment:
But test_utils still uses that if it is set - the master API key thing is just a fallback.